cypress/e2e/3-testing/logout.cy.js

```diff
@@ -0,0 +1,8 @@
+describe('Logout functionality', () => {
+  it('User can log out with the logout button', () => {
+    cy.visit('../../index.html');
+    cy.get('button[data-auth="logout"]').click();
+    cy.url().should('include', 'http://localhost:63185/');
```

Comment:
Local storage token should be null, but this will be true unless you login first